run_tests.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+case "$@" in
+	*-k*) partial_test=1 ;;
+esac
+
+if [ "$partial_test" = 1 ]; then
+	coverage_arg=''
+else
+	# show test coverage for files the dir `augur`
+	coverage_arg='--cov=augur'
+fi
+
+python3 -m pytest -s $coverage_arg "$@"
```

Comment:
I would prefer to keep all pytest configuration settings in one place. The pytest config options (`-s` and `--cov=augur`) can be moved into the existing pytest config file and always executed. Then this run_tests script could be a single line that runs `python -m pytest -c pytest.python3.ini`.

Reply:
:+1: on moving `-s`.

See comment below about omitting `--cov` when `-k` is specified.